### PR TITLE
Fix readme path and add users

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Adds some users to the CLA integration

Also fixes the path of the document

### CODE REVIEW

**Comment by anxolin** [comment]
pls, @alfetopito  let me know if i miss someone from protofire

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           path-to-signatures: 'signatures/version1/cla.json'
-          path-to-cla-document: 'https://github.com/gnosis/cowswap/blob/main/GNOSIS_CLA.md'
+          path-to-cla-document: 'https://github.com/gnosis/cowswap/blob/develop/docs/GNOSIS_CLA.md'
           branch: 'cla-signatures'
-          allowlist: alfetopito,anxolin,W3stside,nenadV91,biocom
+          allowlist: dependabot,mergify,alfetopito,anxolin,W3stside,nenadV91,biocom,alongoni,annamsgeorge,avsavsavs,biocom,c3rnst,cmagan,davidalbela,fedgiac,elena-zh,fleupold,giacomolicari,GabrielCamba,nlordell,ramirotw,josojo
```
Comment (alfetopito) [inline .github/workflows/cla.yml:24]: You've missed:

`henrypalacios` and `matextrem`

BTW, isn't there a way to exclude automatically everyone that for example has write permission?

Comment (anxolin) [reply .github/workflows/cla.yml:24]: Fixed! 

> BTW, isn't there a way to exclude automatically everyone that for example has write permission?
No idea, i assumed not cause this is how safe did it. I didn't read much about the CLA tbh, i didn't want to spend to much on it

**Review by alfetopito — APPROVED** [review]
🤷

**Pull request merged**